dotnet/LADP_EFC/Data/DataContext.cs

```diff
@@ -87,6 +91,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 entity.Property(e => e.Id).HasColumnName("Id").HasColumnType("int");
                 entity.Property(e => e.Name).HasMaxLength(100).HasColumnName("Name").HasColumnType("nvarchar(100)");
             });
+
+            modelBuilder.Entity<Developer>(entity =>
+            {
+                entity.ToTable("DevelopersInfo");
+                entity.HasKey(e => e.DeveloperID).HasName("PK_DevelopersInfo");
+                entity.Property(e => e.DeveloperID).HasColumnName("DeveloperID").HasColumnType("int");
+                entity.Property(e => e.Name).IsRequired().HasMaxLength(255).HasColumnName("Name").HasColumnType("nvarchar(255)");
+                entity.Property(e => e.Role).IsRequired().HasMaxLength(100).HasColumnName("Role").HasColumnType("nvarchar(100)");
+                entity.Property(e => e.PictureUrl).HasColumnName("PictureUrl").HasColumnType("nvarchar(MAX)");
+                entity.Property(e => e.LinkedInUrl).HasColumnName("LinkedInUrl").HasColumnType("nvarchar(MAX)");
+                entity.Property(e => e.GithubUrl).HasColumnName("GithubUrl").HasColumnType("nvarchar(MAX)");
```

Comment:
These column types should match what you have in the database `nvarchar(255)` to avoid errors. `nvarchar(MAX)` should be avoided because of the amount of space it takes in the database. The description in FoodResource will need to be addressed eventually as well in another PR as it uses `nvarchar(MAX)`  as well.
![image](https://github.com/user-attachments/assets/d5a28ba6-3817-4ded-aff9-9571c2b9eb28)

Reply:
Everything else looks good. I think what Jesse said is all that has to be updated.